target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -313,6 +313,12 @@ rp-n53)
 	ucidef_set_led_netdev "eth" "Network" "$board:white:back" "eth0"
 	set_wifi_led "$board:blue:wifi"
 	;;
+rt-ac51u)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```

Comment:
Drop this LED here, it is already handled by diag.sh and should be on after boot.